lib/fastlane/setup/setup_ios.rb

```diff
@@ -52,6 +45,47 @@ def run
       # rubocop:enable Lint/RescueException
     end
 
+    def default_setup(path: nil)
+      copy_existing_files
+      generate_appfile(manually: false)
+      detect_installed_tools # after copying the existing files
+      create_app_if_necessary
+      enable_deliver
+      FileUtils.mkdir(path)
+      generate_fastfile(manually: false)
+      show_analytics
+    end
+
+    def manual_setup(path: nil)
+      copy_existing_files
+      generate_appfile(manually: true)
+      detect_installed_tools # after copying the existing files
+      ask_to_enable_other_tools
+      FileUtils.mkdir(path)
+      generate_fastfile(manually: true)
+      show_analytics
+    end
+
+    def ask_to_enable_other_tools
+      create_app = agree('Would you like to create your app on iTunes Connect and the Developer Portal?', true)
```

Comment:
you're totally right, we should check if its there before asking right? what is the best way to check if the app exists on iTC? i was playing around with using produce and deliver to try to get confirmation that an app exists but didn't know if i was finding exactly what i was looking for.